dev-environment.yml

```diff
@@ -0,0 +1,17 @@
+name: mosaic-dev
+channels: 
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python >= 3.9
+  - cartopy
+  - cmocean
+  - h5netcdf
+  - hdf5
+  - netcdf4
```

Comment:
I would remove these unless `mosaic` really won't work without them.  I think they'll come in through `xarray`, which is how you want them to come in.

Reply:
Oddly, enough these are [optional dependencies](https://docs.xarray.dev/en/stable/getting-started-guide/installing.html) to `xarray`. So we need to explicitly add them to be able to read `netCDF` files. 

I ran into problems when I was testing an initial pass of the dev environment since they weren't explicitly included.

Reply:
Okay, thanks for the explanation there.